demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, name: str = '', contribution: Union[str] = None, description:
         self.pack_dir_path = os.path.join(self.packs_dir_path, self.dir_name)
         if not os.path.isdir(self.pack_dir_path):
             os.makedirs(self.pack_dir_path)
+        self.readme_files: List[str] = []
```

Comment:
add a `Attributes` section to the docstring and explain about that attribute there
see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for example